src/System.Net.Requests/tests/HttpRequestResponseSerializationTest.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization.Formatters.Tests;
+using Xunit;
+
+namespace System.Net.HttpRequest.Tests
+{
+    public partial class HttpRequestResponseSerialization
+    {
+        public readonly static object[][] EchoServers = System.Net.Test.Common.Configuration.Http.EchoServers;
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        [ActiveIssue(13500)]
+        //// TODO: Issue #13500
```

Comment:
It'd be nice to actually implement Timeout properly as part of this PR.  But if for some reason you want to defer that to a separate PR, at least add the override of Timeout, and just have it get/set a field and do argument validation, which will be necessary for making it work properly anyway... then you can save hooking it up to HttpClient.Timeout for a subsequent PR if you want.  That way, you can remove the [ActiveIssue] from this test, though I still think this test is going to fail to due to the issue I mentioned below about Assert.Equal, and marking this as ActiveIssue due to Timeout not being implemented is just covering that up.